<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impact</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/images.qrc">
    <normaloff>:/images2/new/impact.png</normaloff>:/images2/new/impact.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="MainMenu1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <widget class="QMenu" name="File1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Itemexit"/>
   </widget>
   <widget class="QMenu" name="Characterization1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Characterisation</string>
    </property>
    <addaction name="syschar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="Maintenance1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Data tables</string>
    </property>
    <widget class="QMenu" name="menuCrops">
     <property name="title">
      <string>Crops</string>
     </property>
     <property name="icon">
      <iconset resource="images/images.qrc">
       <normaloff>:/images2/new/tomato.png</normaloff>:/images2/new/tomato.png</iconset>
     </property>
     <addaction name="actionCrops"/>
     <addaction name="actionCrop_products"/>
     <addaction name="actionCrop_forages"/>
    </widget>
    <widget class="QMenu" name="menuPastures">
     <property name="title">
      <string>Pastures</string>
     </property>
     <property name="icon">
      <iconset resource="images/images.qrc">
       <normaloff>:/images/grass.png</normaloff>:/images/grass.png</iconset>
     </property>
     <addaction name="actionPastures"/>
     <addaction name="actionPasture_forages"/>
    </widget>
    <widget class="QMenu" name="menuLivestock">
     <property name="title">
      <string>Livestock</string>
     </property>
     <property name="icon">
      <iconset resource="images/images.qrc">
       <normaloff>:/images2/new/live.png</normaloff>:/images2/new/live.png</iconset>
     </property>
     <addaction name="actionLivestock"/>
     <addaction name="actionLivestock_products"/>
    </widget>
    <widget class="QMenu" name="menuActivities">
     <property name="title">
      <string>Activities</string>
     </property>
     <property name="icon">
      <iconset resource="images/images.qrc">
       <normaloff>:/images2/new/activity.png</normaloff>:/images2/new/activity.png</iconset>
     </property>
     <addaction name="actionActivities"/>
     <addaction name="actionActivity_units"/>
    </widget>
    <addaction name="Systemtypes1"/>
    <addaction name="menuCrops"/>
    <addaction name="menuPastures"/>
    <addaction name="menuActivities"/>
    <addaction name="Ferttypes1"/>
    <addaction name="menuLivestock"/>
    <addaction name="Purchasedinputs1"/>
    <addaction name="Householdconsumptionproducts1"/>
    <addaction name="separator"/>
    <addaction name="Countries1"/>
    <addaction name="Soilds1"/>
    <addaction name="Drainage1"/>
    <addaction name="Marketingoutlets1"/>
   </widget>
   <widget class="QMenu" name="Analysetools1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Analysis tools</string>
    </property>
    <addaction name="InputsOutputs1"/>
    <addaction name="Foodsecurity1"/>
   </widget>
   <widget class="QMenu" name="Experiments1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="ProjectSettings"/>
    <addaction name="DBMan"/>
   </widget>
   <widget class="QMenu" name="Help1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="About1"/>
   </widget>
   <addaction name="File1"/>
   <addaction name="Characterization1"/>
   <addaction name="Maintenance1"/>
   <addaction name="Analysetools1"/>
   <addaction name="Experiments1"/>
   <addaction name="Help1"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="Itemexit">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/exit.png</normaloff>:/images2/new/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="syschar">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/systems.png</normaloff>:/images2/new/systems.png</iconset>
   </property>
   <property name="text">
    <string>System characterisation</string>
   </property>
  </action>
  <action name="ExportSystem">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/systemsexp.png</normaloff>:/images2/new/systemsexp.png</iconset>
   </property>
   <property name="text">
    <string>Export systems</string>
   </property>
  </action>
  <action name="ImportSystem">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/systemsimp.png</normaloff>:/images2/new/systemsimp.png</iconset>
   </property>
   <property name="text">
    <string>Import systems</string>
   </property>
  </action>
  <action name="Ferttypes1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/ferttypes2.png</normaloff>:/images2/new/ferttypes2.png</iconset>
   </property>
   <property name="text">
    <string>Fertilisers</string>
   </property>
  </action>
  <action name="Purchasedinputs1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/purchase.png</normaloff>:/images2/new/purchase.png</iconset>
   </property>
   <property name="text">
    <string>Purchased feeds</string>
   </property>
  </action>
  <action name="Householdconsumptionproducts1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/hhcond.png</normaloff>:/images2/new/hhcond.png</iconset>
   </property>
   <property name="text">
    <string>Household consumption products</string>
   </property>
  </action>
  <action name="Countries1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/countries.png</normaloff>:/images2/new/countries.png</iconset>
   </property>
   <property name="text">
    <string>Countries</string>
   </property>
  </action>
  <action name="Soilds1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/soil.png</normaloff>:/images2/new/soil.png</iconset>
   </property>
   <property name="text">
    <string>Soils</string>
   </property>
  </action>
  <action name="Drainage1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/drainage.png</normaloff>:/images2/new/drainage.png</iconset>
   </property>
   <property name="text">
    <string>Drainage</string>
   </property>
  </action>
  <action name="Marketingoutlets1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/marks.png</normaloff>:/images2/new/marks.png</iconset>
   </property>
   <property name="text">
    <string>Marketing outlets</string>
   </property>
  </action>
  <action name="InputsOutputs1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/units.png</normaloff>:/images2/new/units.png</iconset>
   </property>
   <property name="text">
    <string>Cash flow</string>
   </property>
  </action>
  <action name="Foodsecurity1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/foodsec.png</normaloff>:/images2/new/foodsec.png</iconset>
   </property>
   <property name="text">
    <string>Food security</string>
   </property>
  </action>
  <action name="Nutrientflows1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/nutflows.png</normaloff>:/images2/new/nutflows.png</iconset>
   </property>
   <property name="text">
    <string>Nutrient flows</string>
   </property>
  </action>
  <action name="Labourefficiency1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/laboranalysis.png</normaloff>:/images2/new/laboranalysis.png</iconset>
   </property>
   <property name="text">
    <string>Labour efficiency</string>
   </property>
  </action>
  <action name="About1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/help.png</normaloff>:/images2/new/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="ProjectSettings">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/project.png</normaloff>:/images2/project.png</iconset>
   </property>
   <property name="text">
    <string>Project settings</string>
   </property>
  </action>
  <action name="DBMan">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/database.png</normaloff>:/images2/database.png</iconset>
   </property>
   <property name="text">
    <string>Project manager</string>
   </property>
   <property name="toolTip">
    <string>Project manager</string>
   </property>
  </action>
  <action name="actionCrop_products">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/grain.png</normaloff>:/images/grain.png</iconset>
   </property>
   <property name="text">
    <string>Crop products</string>
   </property>
  </action>
  <action name="actionCrop_forages">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/leaves.png</normaloff>:/images/leaves.png</iconset>
   </property>
   <property name="text">
    <string>Crop forages</string>
   </property>
  </action>
  <action name="actionCrops">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/tomato.png</normaloff>:/images2/new/tomato.png</iconset>
   </property>
   <property name="text">
    <string>Crops</string>
   </property>
  </action>
  <action name="actionPastures">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/grass.png</normaloff>:/images/grass.png</iconset>
   </property>
   <property name="text">
    <string>Pastures</string>
   </property>
  </action>
  <action name="actionPasture_forages">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/leaves.png</normaloff>:/images/leaves.png</iconset>
   </property>
   <property name="text">
    <string>Pasture forages</string>
   </property>
  </action>
  <action name="Systemtypes1">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/systtypes.png</normaloff>:/images2/new/systtypes.png</iconset>
   </property>
   <property name="text">
    <string>System types</string>
   </property>
  </action>
  <action name="actionLivestock">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/live.png</normaloff>:/images2/new/live.png</iconset>
   </property>
   <property name="text">
    <string>Livestock</string>
   </property>
  </action>
  <action name="actionLivestock_products">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/eggs.png</normaloff>:/images/eggs.png</iconset>
   </property>
   <property name="text">
    <string>Livestock products</string>
   </property>
  </action>
  <action name="actionActivities">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images2/new/activity.png</normaloff>:/images2/new/activity.png</iconset>
   </property>
   <property name="text">
    <string>Activities</string>
   </property>
  </action>
  <action name="actionActivity_units">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/images/units.png</normaloff>:/images/units.png</iconset>
   </property>
   <property name="text">
    <string>Activity units</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
